.github/workflows/tests.yml

```diff
@@ -0,0 +1,21 @@
+name: CI Tests
+on: [push]
```

Comment:
Do we need to run this on every branch?

Reply:
I believe that is the current behavior for CircleCI and Travis.